Legion/DarkheartThicket/ShadeOfXavius.lua

```diff
@@ -52,6 +54,12 @@ function mod:GrowingParanoiaRemoved(args)
 	self:PrimaryIcon(args.spellId)
 end
 
+function mod:FeedOnTheWeakApplied(args)
+	if self:Me(args.destGUID) or self:Healer() then
+		self:TargetMessage(args.spellId, args.destName, self:Healer() and "Important" or "Personal", "Warning", nil, nil, true)

```

Comment:
No need for that check for the color since `:TargetMessage` defaults to "Personal" if the player is the target.
See https://github.com/BigWigsMods/BigWigs/blob/master/Core/BossPrototype.lua#L1532

Reply:
Okay, fixed.